consent-ws/src/main/java/org/broadinstitute/consent/http/resources/DACUserResource.java

```diff
@@ -98,6 +98,21 @@ public Response update(@Context UriInfo info, Map<String, DACUser> dac, @PathPar
         }
     }
 
+    @PUT
+    @Path("/main_fields/{id}")
```

Comment:
Nit: We generally camel case paths - the only exception I see in the code is `group-names`. We should try to be consistent here. 
```suggestion
    @Path("/mainFields/{id}")
```